src/main/java/com/amazonaws/services/sqs/AmazonSQSRequesterClientBuilder.java

```diff
@@ -16,6 +16,7 @@
     
     private int idleQueueSweepingPeriod = 5;
     private TimeUnit idleQueueSweepingTimeUnit = TimeUnit.MINUTES;
+    private String queueRetentionPeriodSeconds = AmazonSQSTemporaryQueuesClientBuilder.QUEUE_RETENTION_PERIOD_SECONDS_DEFAULT;
```

Comment:
This should definitely be a Long everywhere instead. It was only a String before because it was hard-coded and only every used as a queue attribute, but now we should only be converting to and from strings in the code that reads and writes that queue attribute.

Reply:
Made the change to long